src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -382,8 +406,13 @@ private function isMaxDepthReached(array $attributesMetadata, $class, $attribute
      */
     private function getCacheKey($format, array $context)
     {
+        unset($context['cache_key']); // avoid artifically different keys
```

Comment:
if we would keep this field, every call to this method generates a new key as the previous key is part of what we hash. the correct thing is to not include the previous key in the hash as its not meaningful information.